frontend/src/features/admin-form/settings/components/FormLimitToggle.tsx

```diff
@@ -106,6 +109,8 @@ export const FormLimitToggle = (): JSX.Element => {
   const { data: settings, isLoading: isLoadingSettings } =
     useAdminFormSettings()
 
+  const isMRF = settings?.responseMode === FormResponseMode.Multirespondent
```

Comment:
We usually only capitalize the first letter of an acronym so that variables that are made up of acronyms doesn't become an abomination like `XMLHTTPAsJSON` vs `xmlHttpAsJson`